starkinfra/creditnotepreview/__creditnotepreview.py

```diff
@@ -0,0 +1,78 @@
+    - scheduled [datetime.date, datetime.datetime or string]: date of transfer execution. ex: scheduled=datetime(2020, 3, 10)
+    - tax_id [string]: credit receiver's tax ID (CPF or CNPJ). ex: "20.018.183/0001-80"
+    ## Parameters (conditionally required):
+    - nominal_interest [float]: yearly nominal interest rate of the credit note, in percentage. ex: 12.5
+    - initial_due [datetime.date, datetime.datetime or string]: The date of the first invoice. ex: scheduled=datetime(2020, 3, 10)
+    - count [integer]: The quantity of invoices for payment. ex: 12
+    - invoices [list of Invoice objects]: list of Invoice objects to be created and sent to the credit receiver. ex: invoices=[Invoice(), Invoice()]
+    - initial_amount [integer]: The value of the first invoice in cents. ex: 1234 (= R$12.34)
+    - interval [string]: The interval between invoices. ex: "year", "month"
+    ## Parameters (optional):
+    - rebate_amount [integer, default None]: credit analysis fee deducted from lent amount. ex: rebate_amount=11234 (= R$ 112.34)
+    ## Attributes (return-only):
+    - amount [integer]: CreditNote value in cents. ex: 1234 (= R$ 12.34)
+    - tax_amount [integer]: tax amount included in the CreditNote. ex: 100
+    - interest [float]: yearly effective interest rate of the credit note, in percentage. ex: 12.5
+    """
+
+    def __init__(self, type, nominal_amount, scheduled, tax_id=None, invoices=None, nominal_interest=None,
+                 initial_due=None, count=None, rebate_amount=None, amount=None, initial_amount=None, interest=None,
+                 tax_amount=None, interval=None):
+        SubResource.__init__(self)
+
+        self.type = type
+        self.nominal_amount = nominal_amount
+        self.scheduled = scheduled
+        self.tax_id = tax_id
+        self.invoices = _parse_invoices(invoices) if invoices else None
+        self.nominal_interest = nominal_interest
+        self.initial_due = initial_due
+        self.count = count
+        self.rebate_amount = rebate_amount
+        self.amount = amount
+        self.initial_amount = initial_amount
+        self.interest = interest
+        self.tax_amount = tax_amount
+        self.interval = interval
+
+
+_resource = {"class": CreditNotePreview, "name": "CreditNotePreview"}
+
+
+def _parse_invoices(invoices):
+    parsed_invoices = []
+    for invoice in invoices:
+        if isinstance(invoice, Invoice):
+            parsed_invoices.append(invoice)
+            continue
+        parsed_invoices.append(from_api_json(_invoice_resource, invoice))
+    return parsed_invoices
+
+
+def create(previews, user=None):
+    """# Create CreditNotePreview
+    Send a list of CreditNotePreview objects for send to the Stark Infra API
+    ## Parameters (required):
+    - previews [list of CreditNotePreview objects]: list of CreditNotePreview objects to be send to the API
```

Comment:
- previews [list of CreditNotePreview objects]: list of CreditNotePreview objects to be created in the API